python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/traffic-policies.j2

```diff
@@ -7,207 +7,207 @@
-         destination prefix {{ match.destination.prefixes | join(' ') }}
-{%                         endif %}
-{%                         if match.destination.prefix_lists is arista.avd.defined %}
-         destination prefix field-set {{ match.destination.prefix_lists | join(' ') }}
-{%                         endif %}
-{%                     endif %}
-{#                     -- PROTOCOL MANAGEMENT -- #}
-{%                     if match.protocols is arista.avd.defined %}
-{%                         for protocol in match.protocols %}
-{#                             -- destination port-list -- #}
-{%                             if protocol.dst_port is arista.avd.defined and protocol.protocol | lower in ['tcp','udp'] %}
-{%                                 set protocol_cli = 'protocol ' ~ protocol.protocol %}
-{%                                 if protocol.flags is arista.avd.defined %}
-{%                                     set protocol_cli = protocol_cli ~ ' flags ' ~ protocol.flags | join(' ') %}
-{%                                 endif %}
-{%                                 set protocol_cli = protocol_cli ~ ' destination port ' ~ protocol.dst_port %}
-         {{ protocol_cli }}
-{#                             -- source port-list -- #}
-{%                             elif protocol.src_port is arista.avd.defined and protocol.protocol | lower in ['tcp','udp'] %}
-{%                                 set protocol_cli = 'protocol ' ~ protocol.protocol %}
-{%                                 if protocol.flags is arista.avd.defined %}
-{%                                     set protocol_cli = protocol_cli ~ ' flags ' ~ protocol.flags | join(' ') %}
+{#                 -- SOURCE PREFIX MANAGEMENT -- #}
+{%                 if match.source.prefixes is arista.avd.defined %}
+         source prefix {{ match.source.prefixes | arista.avd.natural_sort | join(' ') }}
+{%                 elif match.source.prefix_lists is arista.avd.defined %}
+         source prefix field-set {{ match.source.prefix_lists | arista.avd.natural_sort | join(' ') }}
+{%                 endif %}
+{#                 -- DESTINATION PREFIX MANAGEMENT -- #}
+{%                 if match.destination.prefixes is arista.avd.defined %}
+         destination prefix {{ match.destination.prefixes | arista.avd.natural_sort | join(' ') }}
+{%                 elif match.destination.prefix_lists is arista.avd.defined %}
+         destination prefix field-set {{ match.destination.prefix_lists | arista.avd.natural_sort | join(' ') }}
+{%                 endif %}
+{#                 -- PROTOCOL MANAGEMENT -- #}
+{%                 if match.protocols is arista.avd.defined %}
+{%                     set bgp_flag = True %}
+{%                     for protocol in match.protocols %}
+{%                         if protocol.protocol in ['neighbors', 'bgp'] and bgp_flag %}
+{%                             if protocol.protocol | lower == 'neighbors' and protocol.enforce_gtsm is arista.avd.defined(true) %}
```

Comment:
Need to handle the case when bgp and neighbors both protocol is defined in input